runtime/js/40_testing.js

```diff
@@ -363,6 +364,12 @@
     }
   }
 
+  // Some resources cannot be explicitly closed and depend on
+  // garbage collection.
+  const resourceAllowList = [
+    "urlResource",
+  ];
```

Comment:
This is really special case... I'm worried this will open doors for more special cases... Maybe we could store it inside `OpState` instead of `ResourceTable`?